src/index.spec.ts

```diff
@@ -224,8 +225,15 @@ describe('EppoJSClient E2E test', () => {
       return mockObfuscatedUfcFlagConfig;
     });
 
+    const mockAssignmentCache = td.object<AssignmentCache>();
+    td.when(mockAssignmentCache.has(td.matchers.anything())).thenReturn(false);
+    td.when(mockAssignmentCache.set(td.matchers.anything())).thenReturn();
+    globalClient.useCustomAssignmentCache(mockAssignmentCache);
+
+    const mockLogger = td.object<IAssignmentLogger>();
+    globalClient.setAssignmentLogger(mockLogger);
```

Comment:
Without these mocks for this test, this test would fail if run along with the test before it.